packages/api-console-gui/src/app/app.component.ts

```diff
@@ -1,31 +1,45 @@
 import { Component, ElementRef } from '@angular/core';
-import { DuiApp } from '@deepkit/desktop-ui';
+import { DuiApp, DuiStyleComponent, WindowComponent, WindowContentComponent, WindowHeaderComponent, WindowToolbarComponent, WindowToolbarContainerComponent } from '@deepkit/desktop-ui';
 import { ControllerClient } from './client';
+import { HeaderLogoComponent, HeaderStatusBarComponent } from '@deepkit/ui-library';
+import { RouterLink, RouterOutlet } from '@angular/router';
 
 @Component({
     selector: 'app-root',
     template: `
-        <dui-window>
-            <dui-window-header size="small">
-                <dui-window-toolbar>
-                    <deepkit-header-logo title="API Console"></deepkit-header-logo>
+      <dui-style />
+      <dui-window normalize-style>
+        <dui-window-header size="small">
+          <dui-window-toolbar>
+            <deepkit-header-logo title="API Console"></deepkit-header-logo>
 
-                    <dui-window-toolbar-container name="main"></dui-window-toolbar-container>
-                    <div class="top-right">
-                        <div>
-                            <a routerLink="/api">OVERVIEW</a>
-                        </div>
-                        <deepkit-header-status-bar></deepkit-header-status-bar>
-                    </div>
-                </dui-window-toolbar>
-            </dui-window-header>
-            <dui-window-content [sidebarVisible]="sidebarVisible" class="no-padding">
-                <router-outlet></router-outlet>
-            </dui-window-content>
-        </dui-window>
+            <dui-window-toolbar-container name="main"></dui-window-toolbar-container>
+            <div class="top-right">
+              <div>
+                <a routerLink="/api">OVERVIEW</a>
+              </div>
+              <deepkit-header-status-bar [client]="client.client"></deepkit-header-status-bar>
+            </div>
+          </dui-window-toolbar>
+        </dui-window-header>
+        <dui-window-content [sidebarVisible]="sidebarVisible" class="no-padding">
+          <router-outlet></router-outlet>
```

Comment:
I encourage consistent convention for self-closing tags:
- <dui-style /> uses self-closing
- <deepkit-header-logo>, <dui-window-toolbar-container>, <deepkit-header-status-bar> and <router-outlet> use open/closing tags convention